packages/lisk-api-client/package.json

```diff
@@ -41,16 +41,20 @@
 		"test:watch:min": "npm run test:watch -- --reporter=min",
 		"test:node":
 			"npm run build:check && BABEL_ENV=buildcheck mocha test --compilers js:babel-register --recursive",
-		"test:browser": "cypress run --env ROOT_DIR=\"${PWD##*/}\"",
-		"serve:browsertest": "http-server browsertest",
+		"pretest:browser":
+			"npm run build:browsertest && http-server -p 8080 browsertest &",
+		"test:browser":
+			"wait-on http://localhost:8080 && cypress run --env ROOT_DIR=\"${PWD##*/}\" && echo 'test-finished'",
+		"posttest:browser": "echo 'posttest' && kill $(lsof -t -i:8080)",
 		"cover":
 			"if [ -z $JENKINS_HOME ]; then npm run cover:local; else npm run cover:ci; fi",
 		"cover:base": "NODE_ENV=test nyc report",
 		"cover:local": "npm run cover:base -- --reporter=html --reporter=text",
 		"cover:ci": "npm run cover:base -- --reporter=text",
 		"build:browsertest":
 			"npm run transpile:browsertest && npm run browserify:browsertest && npm run uglify:browsertest",
-		"postbuild:browsertest": "rm -r browsertest/src browsertest/test",
+		"postbuild:browsertest":
+			"rm -r browsertest.build/src browsertest.build/test",
```

Comment:
Could you create an issue to remove `json` files from prettier and reformat them?